ui/app/adapters/totp-key.js

```diff
@@ -23,65 +22,57 @@ export default ApplicationAdapter.extend({
       response.data.id = name;
       return response;
     });
-  },
+  }
 
   createRecord() {
     return this.createOrUpdate(...arguments);
-  },
-
-  updateRecord() {
```

Comment:
Since keys can only be created, we don't need a shared `createOrUpdate` method and that logic can just be moved to the `createRecord()` method. 

It might also be worth adding a comment above `createRecord` that totp keys can only be generated so there's no update method